aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -23,6 +23,19 @@ module aptos_framework::account {
     friend aptos_framework::resource_account;
     friend aptos_framework::transaction_validation;
 
+    #[event]
+    struct KeyRotation has drop, store {
+        account: address,
+        old_authentication_key: vector<u8>,
+        new_authentication_key: vector<u8>,
+    }
+
+    #[event]
+    struct CoinRegister has drop, store {
```

Comment:
I wonder if we just leave this one out since registration is no longer a required step

Reply:
if nobody uses it, we could omit it. But hard to get feedbacks.
btw, now my question is it seems this PR breaks a lot of tests...

Reply:
It breaks because of this? Or something else

I say leave it out if it doesn't break anything, it is less confusing in the long term

Reply:
not this one. you mean keeping it?

Reply:
Yeah, I meant: does this change right here break a lot of tests? Or did you mean in general this double emission PR break a lot of tests? Haha

And then:

I think we should leave this:
```rust
#[event]
struct CoinRegister has drop, store {
 ...
}
```
out, because we don't recommend people use the register flow anymore